src/arch.rs

```diff
@@ -1,8 +1,9 @@
-#![no_std]
-
 pub mod interrupts;
 pub mod io;
 pub mod mm;
+pub mod ia32;
```

Comment:
As discussed, there should be a `#[cfg(...)]`

Reply:
Fixed in [ae3d773](https://github.com/P-E-P/flint/pull/13/commits/ae3d773b58f534dd1fb5e71f59c06f70d09028ef).